.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+## Bug Description
+<!--
+Please provide a clear and concise description of what the bug is.
+
+What did you expect to happen? What actually happened?
+
+For bugs with fixers: How the code was fixed? How did you expect the code to be fixed?
+-->
+
+## Minimal Code Snippet
+<!-- Please provide example code that demonstrates the issue. Do NOT paste screenshots of code! -->
+
+```
```

Comment:
The backticks + php ( or js or css ) will create the syntax highlighting. Without the language indication, it will just be a plain grey code block.